src/pushpa/Assignment53.java

```diff
@@ -0,0 +1,38 @@
+//Write a program to return unique vowels included in the string, ensure sequence is maintained.
+//String str = "aakaniksha eaep dube";
+//output : aieu
+//
+//String getVowels(String str){
+//	String output = "";
+//	while(str.length() != 0){
+//		
+//	}	
+//}
+
+package pushpa;
+
+public class Assignment53 {
+
+	public String getUniqueVowels(String input) {
+		String uniqueVowels = "";
+		int i = 0;
+		while (i < input.length()) {
+			Character ch = input.charAt(i);
+			if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u' || ch == 'A' || ch == 'E' || ch == 'I'
```

Comment:
Instead of put so many conditions, you can just convert string into either in lowercase or uppercase and then just need to put 5 conditions